book/website/communication/peer-review/peer-review-conflicts.md

```diff
@@ -0,0 +1,83 @@
+(cm-pr-conflicts)=
+# Conflict of Interest
+
+## Context
+Imagine if a researcher for a national institute with a secondment at a climate consulting business publishes a paper about the categorisation of a hurricane speed and suggests a re-categorisation plus a novel fast-track detection with a data-driven method. 
+Then, we later find out that the editor of the paper also works for the same consulting company and the solution being suggested uses the editors’ patent. 
+And that the researcher of the paper never mentions as part of a standard disclosure that the company (and potentially the author) has a major financial stake in the research outcomes. 
+Believe it or not this is a common problem in scientific publishing, and raises the relevance of declaring the so-called conflict of interest {cite:ps}`weinkle2024coi`.
+
+## What?
+To maintain trust and integrity of scientific publishing in the global era, it is imperative to identify conflicting biases (conscious or unconscious) across all participants in the peer-reviewed publication process - including peer reviewers, editors, and the editorial board members of the journals. 
+A “conflict of interest” or “competing interest” (COI) refers to situations that are, or could be perceived to interfere with the objective presentation, review, or publication of a piece of work {cite:ps}`DeGruyter2024coi`.
+
+## Types of conflicts
+COIs can be divided into main two groups, financial and non-financial {cite:ps}`taylorfrancis2024coi`.
+
+### Examples of financial conflict of interest: 
+* Employment or voluntary involvement
+* Collaborations with advocacy groups related to the content of the research output
+* Grants to the author or organisation
+* Personal fees (honoraria, consulting fees, lecture fees)
+* Intellectual property (patents, copyrights, royalties)
+* Stock or share ownership
+* Benefits related to the development of products
+
+### Examples of non-financial conflict of interest: 
+* Gifts (equipment, tools, software)
+* Access to data repositories
+* Holding a position on boards
+* Close relationships with editors at the journal
+* Competing interests (personal, political, religious, academic)
+* Involvement in legal action (paper retraction)
+
+## Use cases
+We present below some use cases where the scientific and public communities/initiatives have flagged potential COI, and how the editorial team has handled the situation.
+
+### Examples in Peer Review: 
+The [Committee on Publication Ethics (COPE)](https://publicationethics.org) provides a diverse number of anonymized cases with COI in scientific publications. 
+COPE Forum is a useful platform to discuss a publication ethics case with the wider scientific community with different anonymized use cases available [here](https://publicationethics.org/guidance/Case?t=conflict&classification%5B%5D=2774&sort=date). 
+We present below one of the most recent resolved cases {cite:ps}`cope2023coi`:
```

Comment:
I'm not sure what the Turing Way's policy on this is, but since the below section is copied (almost) verbatim from the COPE Forum, should it be block quoted?

Reply:
@aleesteele may I ask which is the best practice when copying external use cases? shall I rephrase it?